src/generators/template.txt

```diff
@@ -0,0 +1,15 @@
+##########################################################################
```

Comment:
I think the names of the templates should be "tsv_header_template.txt" and "vcf_header_template.txt" to make it more explcit what the templates contain. Otherwise I approve of the pull request.

Reply:
I changed the names in the last commit.